cmd/kube-controller-manager/app/options/attachdetachcontroller.go

```diff
@@ -37,6 +37,7 @@ func (o *AttachDetachControllerOptions) AddFlags(fs *pflag.FlagSet) {
 
 	fs.BoolVar(&o.DisableAttachDetachReconcilerSync, "disable-attach-detach-reconcile-sync", false, "Disable volume attach detach reconciler sync. Disabling this may cause volumes to be mismatched with pods. Use wisely.")
 	fs.DurationVar(&o.ReconcilerSyncLoopPeriod.Duration, "attach-detach-reconcile-sync-period", o.ReconcilerSyncLoopPeriod.Duration, "The reconciler sync wait time between volume attach detach. This duration must be larger than one second, and increasing this value from the default may allow for volumes to be mismatched with pods.")
+	fs.BoolVar(&o.DisableForceDetachOnUnmount, "disable-force-detach", false, "Prevent force detaching volumes based on maximum unmount time and node status. If enabled, the non-graceful node shutdown feature must be used to recover from node failure (see https://kubernetes.io/blog/2023/08/16/kubernetes-1-28-non-graceful-node-shutdown-ga/). If enabled and a pod must be forcibly terminated at the risk of corruption, then the appropriate VolumeAttachment object (see here: https://kubernetes.io/docs/reference/kubernetes-api/config-and-storage-resources/volume-attachment-v1/) must be deleted.")
```

Comment:
This belongs to a separate PR

Reply:
ACK; I think I messed something up in the import process...

Reply:
I've reverted the offending commit; hopefully that was the correct way to address this?

Reply:
Try doing a [interactive rebase](https://git-scm.com/book/en/v2/Git-Tools-Rewriting-History) and delete the commits. Create a new branch to save your commits in case something goes wrong with the rebase.

Reply:
Gotcha; I'll push in a bit...

Reply:
Just pushed; I had to do a forced update but it looks like it worked!